build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -685,7 +688,8 @@ async function loadData(dataFile: string, onlyInternal: boolean): Promise<MapDat
 			addUniqueNamedExportsToMap(
 				[member],
 				entry,
-				onlyInternal && (level === ApiLevel.beta || level === ApiLevel.alpha)
+				onlyInternal &&
+					(level === ApiLevel.beta || level === ApiLevel.alpha || level === ApiLevel.legacy)
```

Comment:
nit: not sure how it affects perf, but a pattern I like (picked it up from Python, where it's idiomatic) is:

```suggestion
				onlyInternal &&
					([ApiLevel.beta, ApiLevel.alpha, ApiLevel.legacy].includes(level))
```

It's especially nice when adding/removing conditions, and I find it more readable, but opinions vary.

Reply:
`includes` is funny. Typescript knows the the types of elements in array. So it would see that as `("beta" | "alpha" | "legacy")[]` and it knows that `level` may have a different value. Typescript flags potential misuse, which I think is a mistake when there are only literals in type union and the query uses the enclosing type. Anyway the alternative that is to place those in a constant array, type it generically, and use a helper function that does type assertion. And if you were going to have a constant array, you probably want a constant Set instead. :)
In this case we don't need the narrowing but it isn't as simple as suggested.
The result if applied:
```
Argument of type 'ApiLevel' is not assignable to parameter of type '"beta" | "alpha" | "legacy"'.
  Type '"public"' is not assignable to type '"beta" | "alpha" | "legacy"'.ts(2345)
```